.github/workflows/ci.yml

```diff
@@ -29,8 +29,8 @@ jobs:
           set -xe
           python -VV
           python -m site
-          python -m pip install --upgrade pip setuptools wheel
-          python -m pip install --upgrade coverage[toml] virtualenv tox tox-gh-actions
+          python -m pip install --upgrade \
```

Comment:
Let's make this all one line since we are not picky about line length and it improves readability.

Reply:
Shrinked to one line